packages/docusaurus/src/commands/unknown.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {CommanderStatic} from 'commander';
+import {loadContext, loadPluginConfigs} from '../server';
+import {initPlugins} from '../server/plugins/init';
+
+export function unknownCommand(cli: CommanderStatic, siteDir: string): void {
+  const context = loadContext(siteDir);
+  const pluginConfigs = loadPluginConfigs(context);
+  const plugins = initPlugins({pluginConfigs, context});
```

Comment:
Should we init our own plugins after 3rd party ones so that 3rd parties cannot overwrite our ones? Not sure whether it's the good thing to do.

Reply:
they cannot overwrite it because if user run known command, the extendCli lifecycle will never hit.

example: when user run “yarn docusaurus swizzle”, swizzle is a known command so “unknownCommand(cli)” function will never hit.

when user run “yarn docusaurua roll”, roll is an unknown command so “unknownCommand(cli)” is invoked so that extendCli can be run on all plugins.

The rest is being done by the commander package.

Hence the naming of unknownCommand 😂

Reply:
Hmm oh ya good point. I still think unknown is not the best name. I suggest unrecognized/unregistered

Reply:
externalCommand

we call the known one “internal”. haha